crypto/engine/eng_linux_getrandom.c

```diff
@@ -0,0 +1,150 @@
+#include <openssl/opensslconf.h>
+
```

Comment:
This is way cool and important, but seems like a new feature.  The RNG in OpenSSL needs work -- heck, we created a special mailing list to talk about it -- but nothing was done in time for 1.1 :(